charts/enterprise-metrics/templates/query-frontend-dep.yaml

```
@@ -80,6 +80,9 @@ spec:
               mountPath: /license
             - name: config
               mountPath: /etc/enterprise-metrics
+            - name: storage
+              mountPath: /data
+              subPath: {{ .Values.query_frontend.persistence.subPath }}
```

I think it makes sense to only conditionally set the subPath:

```suggestion
              {{- if .Values.query_frontend.persistence.subPath }}
              subPath: {{ .Values.query_frontend.persistence.subPath }}
              {{- end }}
```